renderer/components/Directory.jsx

```diff
@@ -1,76 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
+import cx from 'classnames';
 import File from './File.jsx';
 import CreateMenu from './CreateMenu.jsx';
 import CreateForm from './CreateForm.jsx';
 import RenameForm from './RenameForm.jsx';
 
-export default class Directory extends React.Component {
-  constructor() {
-    super();
-  }
+export default class Directory extends Component {
+  handleListItemClick = proxyEvent => (
```

Comment:
@isnifer Probably if we can wrap the arrow function with the round parenthesis,
```handleListItemClick = (proxyEvent) => {}```
 it would be more readable in terms of both classical and latest way of JavaScript coding pattern. Let me know your thoughts on this.